week-2-practise/level-3/6-map.js

```diff
@@ -11,16 +11,18 @@ function tidyUpString(str) {
 }
 
 function captialise(str) {
-  // complete this function
+  str = str.split("");
+  str[0] = str[0].toUpperCase();
+  return str.join("") // complete this function
 }
 
 var mentors = ["/Daniel ", "irina ", " Gordon", "ashleigh "];
-var mentorsTidyAndCapitalised;
+var mentorsTidyAndCapitalised = (mentors.map(tidyUpString)).map(captialise);
```

Comment:
Great use of chaining! Can you explain why that works?
You also don't need the extra brackets here :)